### PR TITLE
Fingerless Wool Gloves

### PR DESCRIPTION
**Pull request by SomeGuyIGues**
#### Summary
Content "Added Fingerless Wool Gloves, in XL and XS sizes and recipes for it. Additionally, renamed the Fingerless Gloves to Fingerless Leather Gloves."

#### Purpose of change

The wool gloves hasn't received nearly as much love as its other siblings. The leather gloves, survivor gloves and even the denim gloves have a fingerless variant. I intend to change that.

#### Describe the solution

- Introducing the "Fingerless Wool Gloves." Providing significantly less encumbrance, but less warmth in exchange.
- New recipes to make the item as well as its XL and XS variants.
- Changed the name of the fingerless gloves to the fingerless leather gloves to make it more obvious what the material is.
- The item can be naturally found in the world.

#### Describe alternatives you've considered

- Don't introduce the item and recipe to some degree or altogether, if you hate being warm in style.
- The fingerless wool gloves provides 40 warmth and 7 encumbrance. These values could be changed, but I think they are reasonable considering I own a pair and can interact with my environmental with minimal difficulties.
- I chose 83 yarn for the fingerless wool gloves because it covers up 83.5% of your hands, round down, it seems to make sense to me. However, I didn't actually run the numbers, and this could be changed.

#### Testing

- You can craft the fingerless wool gloves through two recipes, and the XL and XS fingerless wool gloves by cutting a XL and XS wool gloves' tips off.
- The item spawns in the game, after a ton of zombie murdering. I however couldn't find one in a house during my testing. This is likely because I made the item pretty rare to come across, as I assume many would not want to buy wool gloves without fingers.
- Pairs of fingerless gloves are now pairs of fingerless leather gloves.
- Game runs.

#### Additional context

I notice during my play testing that wool gloves when cut up would give too much felt patches. These felt patches can be turned into wool staples, with each being able to be turned into 10 yarn. This wouldn't be an issue if the gloves didn't get cut into disproportionately large amounts. The normal wool gloves gave me 30 felt patches the last time I tested, for a total of 300 yarn if I were to waste my time crafting it.

Also the XL wool gloves do not provide an encumbrance penalty to normal-sized survivors, and due to how I made the XL fingerless wool gloves, they also don't provide an encumbrance penalty. They are 40% bigger than normal gloves, this is dumb.

However, I think that is outside the scope of my PR. I _may_ attempt to fix this, but no promises.

### CODE REVIEW

File: data/json/items/armor/gloves.json [hunk 1]
```diff
@@ -1020,6 +1020,54 @@
...
+      { "encumbrance": 7,
+        "coverage": 75,
+        "covers": [ "hand_l", "hand_r" ],
+        "specifically_covers": [ "hand_wrist_l", "hand_wrist_r", "hand_back_l", "hand_back_r" ]
+      },
+      { "encumbrance": 7, "coverage": 100, "covers": [ "hand_l", "hand_r" ], "specifically_covers": [ "hand_palm_l", "hand_palm_r" ] },
+      {
+        "coverage": 50,
+        "covers": [ "hand_l", "hand_r" ],
+        "specifically_covers": [ "hand_fingers_l", "hand_fingers_r" ]
```
Comment (XygenSS) [inline data/json/items/armor/gloves.json:1050]: They shouldn't be covering the fingers right?

File: data/json/recipes/armor/hands.json
Comment (XygenSS) [inline data/json/recipes/armor/hands.json:1]: You also need a recipe where you take an existing pair of wool gloves and just cut off the fingers.

Comment (SomeGuyIGues) [reply data/json/recipes/armor/hands.json:1]: I thought I did, it's in hands.json

**Comment by XygenSS** [comment]
IIRC 1 hand encumb = 1% slower crafting, the encumbrance needs to be lower for it to be more attractive as crafting gloves imo.

File: data/json/recipes/armor/hands.json [hunk 2]
```diff
@@ -827,6 +827,59 @@
     "time": "6 h 45 m",
     "using": [ [ "tailoring_wool_knitting", 140 ] ]
   },
+  {
+    "result": "gloves_wool_fingerless",
+    "id_suffix": "from_patchwork",
```
Comment (BalthazarArgall) [inline data/json/recipes/armor/hands.json:832]: You don't need an id_suffix for the "base recipe", it's used to differentiate multiple recipes for the same item.

```suggestion
```

**Review by BalthazarArgall — CHANGES_REQUESTED** [review]
This is what I spotted. Also please be careful about formatting issues that may arise from committing from suggestions, you may have to relint the files. I didn't review stuff outside of the items and recipes but you may want to make sure the XS and XL versions are spawning porperly.

**Comment by BalthazarArgall** [comment]
If you feel that your PR is ready you can open it, the tests will reset anyway. And it looks fine, I'm assuming you did your due diligence and everything outside items/recipes is fine, stuff like itemgroups etc. of course.

**Pull request merged**